doc/classes/CharacterBody2D.xml

```diff
@@ -138,20 +162,23 @@
 			As long as the snapping vector is in contact with the ground and the body moves against `up_direction`, the body will remain attached to the surface. Snapping is not applied if the body moves along `up_direction`, so it will be able to detach from the ground when jumping.
 		</member>
 		<member name="floor_stop_on_slope" type="bool" setter="set_floor_stop_on_slope_enabled" getter="is_floor_stop_on_slope_enabled" default="true">
-			If [code]true[/code], the body will not slide on floor's slopes when you include gravity in [code]linear_velocity[/code] when calling [method move_and_slide] and the body is standing still.
+			If [code]true[/code], the body will not slide on floor's slopes when you include gravity in [code]motion_velocity[/code] when calling [method move_and_slide] and the body is standing still.
```

Comment:
When all the recent changes, applying gravity shouldn't influence stop and slope anymore, right?
```suggestion
			If [code]true[/code], the body will not slide on floor's slopes when calling [method move_and_slide] and the body is standing still.
```

Reply:
You will slide if a force pushes you down, so with gravity on floor you will slide.

Even if you don't apply the gravity on the floor (as we can now), you will slide when your velocity push you down (fall).